src/ansys/pyensight/core/utils/omniverse_glb_server.py

```diff
@@ -359,7 +359,13 @@ def upload_file(self, glb_filename: str, timeline: List[float] = [0.0, 0.0]) ->
             # Walk texture nodes -> DSG Variable buffers
             for tex_idx, texture in enumerate(self._gltf.textures):
                 image = self._gltf.images[texture.source]
-                raw_png = self._gltf.get_data_from_buffer_uri(image.uri)
+                if image.uri is None:

```

Comment:
For EnSight generated GLB files, the 1D texture can be inline with the data blob.  In this case, there is no image.url and the buffer view is used to get the PNG payload.